templates/system.json

```diff
@@ -114,6 +114,40 @@
         "fields": "id,login,ringId"
       },
       "url": "api/users"
+    },
+    "workitems-delete": {
```

Comment:
If the datatype is read-only you don't need these operations that only are used in the share pipe (Update/Insert/Delete). In this case you also don't need lookup, but IMO that is more reasonable to leave in.